tests/python/unittest/test_target_target.py

```diff
@@ -76,6 +76,14 @@ def test_target_string_parse():
     assert tvm.target.arm_cpu().device_name == "arm_cpu"
 
 
+def test_target_string_with_spaces():
+    target = tvm.target.Target(
+        "vulkan -device_name=Name\ of\ GPU\ with\ spaces -device_type=discrete"
```

Comment:
In tvmc we faced this challenge of having substrings within target(s) (see [relevant tests](https://github.com/apache/tvm/blob/81480287a891f07f3939b98cc57fad39e217f317/tests/python/driver/tvmc/test_tvmc_common.py#L268-L281)), one example is `llvm` arguments. The decision there was to go with nested quotes, similar to Python. Have you explored this possibility in here?

That would make it look like:
```
"vulkan -device_name='Name of GPU with spaces' -device_type=discrete"
```

I feel that is looks a bit more natural to type in nested quotes rather than preceding everything with a backslash. Also, I think it would be aspirational to offer a similar experience (where possible), when using API and command line.

What do you think?

Reply:
I'd be down for either, and agree that the nested quotes are an easier way to read it.  Consistency with how tvmc handles spaces is also something that I want to maintain, so I'll update this PR to use the same convention.

Reply:
And updated.